tools/topology/topology2/cavs-benchmark-hda.conf

```diff
@@ -284,4 +285,20 @@ IncludeByKey.BENCH_CONFIG {
 	"gain32" {
 		<include/bench/gain_s32.conf>
 	}
+
+	#
+	# IGO NR component
+	#
+
+	"igo_nr16" {
+		<include/bench/igo_nr_s16.conf>
+	}
+
+	"igo_nr24" {
+		<include/bench/igo_nr_s24.conf>
+	}
+
+	"igo_nr32" {
+		<include/bench/igo_nr_s32.conf>
+	}
```

Comment:
possible combine and use one conf for 16/24/32? all three files are short and similar. @ranj063 @singalsu

Reply:
This is automatically generated "code" and if some format is not supported by a component, I just leave out from commit the unused files e.g. aria. In this case I prefer the small test topology .conf building blocks. If a change is I change the generator bench_comp_generate.sh and re-run it for the component.